util/network/src/lib.rs

```diff
@@ -54,6 +54,9 @@
 //! }
 //! ```
 
+//TODO: use Poll from mio
+#![allow(deprecated)]
```

Comment:
how about limit this allowance to deprecated mio imports only?
this todo might live for ages there